packages/mgt-components/src/components/mgt-todo/mgt-todo.ts

```diff
@@ -340,25 +358,46 @@ export class MgtTodo extends MgtTasksBase {
       `
       : html``;
 
-    const taskDeleteTemplate = this.readOnly
-      ? html``
-      : html`
-        <fluent-button class="task-delete"
-          @click="${() => this.removeTask(task.id)}"
-          aria-label="${this.strings.deleteTaskLabel}"
-        >
-          ${getSvg(SvgIcon.Delete)}
-        </fluent-button>
-      `;
+    const taskOptions = this.readOnly
+      ? nothing
+      : mgtHtml`
+          <mgt-dot-options
+            class="dot-options"
+            .options="${{
+              [this.strings.editTaskOption]: () => this.updateTask(task),
```

Comment:
I think that this action would cause the selected task to be rendered in edit mode i.e. using a text input for the name of the task, not showing a complete/incomplete indicator and put focus in the name input.